### PR TITLE
Add initial amp-jwplayer sample

### PR DESCRIPTION
**Pull request by jmadler**
#252

### CODE REVIEW

File: src/20_Components/amp-jwplayer.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  #### Introduction
+
+  Embed jwplayer videos in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
```
Comment (kul3r4) [inline src/20_Components/amp-jwplayer.html]: amp-vimeo -> amp-jwplayer

**Comment by jmadler** [comment]
PTAL

**Comment by kul3r4** [comment]
LGTM, could you squash the commits?

**Comment by cramforce** [comment]
Total aside since @kul3r4 asked for squashing: I switched this GH project from allowing both to "squash merging only", so this happens automatically when you press the big green button and you don't need to ask people. Let me know if anyone doesn't like that :)

**Comment by kul3r4** [comment]
@cramforce thanks!

**Pull request merged**